packages/discovery-react-components/package.json

```diff
@@ -73,7 +73,7 @@
     "rollup": "^2.57.0",
     "rollup-plugin-string": "^3.0.0",
     "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.30.0",
+    "rollup-plugin-typescript2": "^0.34.1",
```

Comment:
This change made TS be a little stricter, resulting in typing fixes in several test files (below).